<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterfaceClass</class>
 <widget class="QMainWindow" name="InterfaceClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Interface</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelNbControleur">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre de contrôleurs</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTailleTrain">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Taille du Train</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelClasse">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Classe</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPopulation">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Population</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPolice">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Présence de la Police</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPeriode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Période de la journée</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelWeekEnd">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Week End</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelVacances">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Vacances</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOrigine">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Origine</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelGenre">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>90</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Genre</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAge">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Age</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelRetard">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Retard</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxNbControleur">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8+</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBoxTailleTrain">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Court</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Moyen</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Long</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBoxClasse">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1ère</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2éme</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBoxPopulation">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Désert</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Faible</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Moyen</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Rempli</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bondé</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBoxPolice">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Oui</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Non</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBoxPeriode">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>150</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Creuse</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Pré/Post Rush</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Rush</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBoxWeekEnd">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>0</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Oui</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Non</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBoxVacance">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Oui</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Non</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBoxOrigine">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Européenne</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Africaine</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Arabe</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Asiatique</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBoxGenre">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>90</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Homme</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Femme</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBoxAge">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Enfant</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Djeunz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Adulte</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Vieux</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBoxRetard">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>150</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0 à 5 min</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5 à 15 min</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15 min et +</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelCheck">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Peut être...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonCheck">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>321</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Dois je prendre le train ?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonAccept">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Evidemment</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonRefuse">
    <property name="geometry">
     <rect>
      <x>234</x>
      <y>230</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Absolument pas</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="interface.qrc"/>
 </resources>
 <connections/>
</ui>
